jenkins/release-tag/release-tag.jenkinsfile

```diff
@@ -17,16 +17,21 @@ pipeline {
             description: 'Version of the release.',
             trim: true
         )
+        choice(
+            choices: ['opensearch', 'opensearch-dashboards'],
+            name: 'PRODUCT',
+            description: 'Product for tag creation.'
+        )
     }
     environment {
-        BUILD_MANIFEST = "opensearch-$VERSION/manifest.yml"
+        BUILD_MANIFEST = "$PRODUCT-$VERSION/manifest.yml"
```

Comment:
Lets always use 'Distribution' instead of 'bundle'

Reply:
> Lets always use 'Distribution' instead of 'bundle'

Bundle Manifest is the term we use in the python lib, using distribution would be confusing even more.

Reply:
And yes this should be called bundle manifest.

Reply:
This is actually used by the jenkins library `BuildManifest`. That's why I have the parameter named `BUILD_MANIFEST`. I will change them all.

Reply:
Its a nit. Also its not an input param but environment variable, just changing here should be fine.

Reply:
Changed in the new commint.